scripts/build_ios.sh

```diff
@@ -9,4 +16,10 @@ echo "Copying frameworks"
 cp -R "${SCRIPT_DIR}/Carthage/Build/iOS/AirshipCore.framework" "${SCRIPT_DIR}/platform/"
 cp -R "${SCRIPT_DIR}/Carthage/Build/iOS/AirshipMessageCenter.framework" "${SCRIPT_DIR}/platform/"
 
-echo "Done"
+echo "Building iOS"
+npx appc run -p ios --build-only
+
+echo "Finished building iOS"
+
+cd -
```

Comment:
I like to be explicit in these, in case some other `cd` creeps into the script.
```suggestion
cd $ROOT_PATH
```